lib/nylas/draft.rb

```diff
@@ -39,6 +39,7 @@ class Draft
     transfer :api, to: %i[events files folder labels]
 
     def update(**data)
+      self.files = data[:files] if data[:files]
```

Comment:
Be sure to test both forks of conditionals when you introduce them into code.

Reply:
We had test cases but they've been further clarified to test for this specific use case.